src/Emu/Metadata/FrontierLabs/LogFileExtractor.cs

```diff
@@ -11,6 +11,9 @@ namespace Emu.Metadata.FrontierLabs
 
     public class LogFileExtractor : IMetadataOperation
     {
+        public const int MegabyteConversion = 1048576;
+        public const int KilobyteConversion = 1024;
```

Comment:
These are incorrect?. Normally these are the correct values to use, but the units you're converting are already measured as <x>bytes.  e.g. these would be correct converting from decimal units to bytes.

I double checked, so I think I'm correct here... the numbers in Fixtures.yaml don't make sense

![image](https://user-images.githubusercontent.com/1452968/178862271-284ef130-2ff5-44ff-97ec-77649d58cc85.png)

Whereas, simply scaling magnitudes:

![image](https://user-images.githubusercontent.com/1452968/178862349-5434d1cd-adc5-4afd-b699-1cf9d0f47716.png)

![image](https://user-images.githubusercontent.com/1452968/178862211-5a11ebbf-c4fe-40c4-8ce2-016f342947db.png)

Reply:
It should be noted that SD uses MB/s (the base 10 scale) everywhere as opposed to MiB/s (the base 2 scale): https://www.sdcard.org/developers/sd-standard-overview/bus-speed-default-speed-high-speed-uhs-sd-express/

I'm fairly sure I'm correct....